### PR TITLE
Update popup edit forms to match design guidelines

### PR DESCRIPTION
**Pull request by epetrow**
Part of: https://github.com/telerik/web-components-planning/issues/137
Closes: https://github.com/telerik/kendo-themes-private/issues/18

### CODE REVIEW

File: packages/html/src/scheduler/tests/scheduler-edit-dialog.tsx [hunk 1]
```diff
@@ -32,76 +32,108 @@ root.render(
...
-                        <FormField label=" " editor={
-                            <>
-                                <Checkbox id="k-is-allday-chkbox" />
-                                <label htmlFor="k-is-allday-chkbox" className="k-checkbox-label">All Day Event</label>
-                            </>
-                        } />
+                        </div> */}
+                        <FormField label="Start" editor={ <DateTimePicker placeholder="dd/MM/yyyy hh:mm AM" /> }/>
+                        {/* <div className="k-form-field">
+                            <div className="k-label k-form-label">
```
Comment (TeyaVes) [inline packages/html/src/scheduler/tests/scheduler-edit-dialog.tsx:56]: why do we need the commented sections here?

**Review by TeyaVes — COMMENTED** [review]
And one more thing I noticed:
-the grid-editing popup is now add item popup, shall it be like this?

**Review by TeyaVes — APPROVED** [review]

File: tests/scheduler/scheduler-edit-dialog.html [hunk 2]
```diff
@@ -118,72 +104,6 @@
...
-                                                    </button>
-                                                    <button class="k-spinner-decrease k-button k-button-md k-button-solid k-button-solid-base k-icon-button">
-                                                        <span class="k-button-icon k-icon k-i-arrow-s"></span>
-                                                    </button>
-                                                </span>
-                                            </span>
-                                            <span>day(s)</span>
-                                        </div>
-                                    </div>
-                                </div>
```
Comment (Juveniel) [inline tests/scheduler/scheduler-edit-dialog.html:140]: we need to have the reference for the recurrence editors as well, please bring them back

**Review by Juveniel — CHANGES_REQUESTED** [review]
LGTM, please address comments so we can proceed and merge this

File: tests/editor/editor-image-editor.html [hunk 3]
```diff
@@ -29,7 +29,7 @@
                     </div>
                 </div>
                 <div class="k-window-content">
-                    <div class="k-form k-form-md k-form-horizontal">
+                    <div class="k-form k-form-md k-form-vertical">
```
[the remark this answers is not in the document]
Comment (epetrow) [reply tests/editor/editor-image-editor.html:32]: Yes, according to the Design guidelines for the forms they should have vertical orientation.

**Comment by Juveniel** [comment]
@epetrow, please verify also that the titles of the Gantt, Grid, TreeList and Scheduler edit/add popups matches the guidelines from [here](https://github.com/telerik/web-components-ux/issues/408#issuecomment-1327396271) e.g:
  *  For Grid and Treelist the titles are: "Add item" and "Edit item";
  *  For Gantt the titles are: "Add task" and "Edit task";
  *  For Scheduler the titles are: "Add event" and "Edit event".

**Review by Juveniel — APPROVED** [review]

**Pull request merged**